README.md

```diff
@@ -18,6 +16,24 @@ You will be graded based on the following have been met:
 
 ----
 
+## Prerequisites/Environment Setup
+
+### Installing Node
+
+The full stack test is developed using Node, using the following versions:
+
+  **Node:** `LTS/Erbium (^12.13.0)`
+
+  **npm:** `^6.12.0`
+
+If you use the latest versions of node and npm its recommended to use [nvm](https://github.com/nvm-sh/nvm) or [nave](https://github.com/isaacs/nave) to manage your Node environment on **Unix** systems. If you use **Windows** then its recommended to use [nvm-windows](https://github.com/coreybutler/nvm-windows) to manage your Node environment.
+
+To install the correct npm version, use `npm install --global npm@^6.12.0`.
```

Comment:
This is done as part of nvm, so I think it confuses things to have this step

Reply:
Ah yes so actually it might be worth saying: `nvm install lts/erbium`? As just using `nvm use` will throw an error saying the version is not installed use `nvm install lts/erbium` to install